lib/HueSensor.js

```diff
@@ -971,10 +973,39 @@ function HueSensor (accessory, id, obj) {
         key: 'temperature',
         name: 'temperature',
         unit: '°C',
-        history: 'weather',
+        history: this.obj.modelid === 'lumi.airmonitor.acn01' ? 'room2' : 'weather',
```

Comment:
That'll work for now, but won't scale well when adding more Eve-Room-like sensors in the future.